server/aws-lsp-codewhisperer/src/language-server/configuration/qConfigurationServer.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    CancellationToken,
+    CredentialsProvider,
+    GetConfigurationFromServerParams,
+    InitializeParams,
+    Server,
+} from '@aws/language-server-runtimes/server-interface'
+import { CodeWhispererServiceToken } from '../codeWhispererService'
+import { getUserAgent } from '../utils'
+
+// The configuration section that the server will register and listen to
+export const Q_CONFIGURATION_SECTION = 'aws.q'
```

Comment:
Could you add documentation for `QConfigurationServerToken` to README with an configuration key it supports and data type of response it provides? 
Also you can define a Typescript type for `onGetConfigurationFromServer` response, and it will serve as a documentation as well.